src/Api/Entity/Payment.php

```diff
@@ -43,6 +44,9 @@ class Payment extends AbstractEntity
 	/** @var string */
 	protected $lang;
 
+	/** @var ElectronicPaymentRegister */
+	protected $electronicPaymentRegister;
```

Comment:
Trosku bojuju s tim nazvem. Co to nazvat zkratka `$eet` a bylo by. Stejne tak entitu.